script/elevate-cpanel.PL

```diff
@@ -387,6 +387,7 @@ sub run ( $pkg, @args ) {
 
     return $self->do_cleanup()      if $self->getopt('clean');
     return $self->blockers->check() if defined $self->getopt('check');
+    die;
```

Comment:
was this a diagnostic check? What's catching this eval?

Reply:
Yeah, I thought I had gotten that, but apparently not. I'll get that fixed up.